src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Mappers/Apprentice/DetailsViewModelMapper.cs

```diff
@@ -140,7 +134,8 @@ private static PendingPriceChange Map(GetManageApprenticeshipDetailsResponse.Pen
             {
                 Cost = priceChangeDetails.Cost,
                 EndPointAssessmentPrice = priceChangeDetails.EndPointAssessmentPrice,
-                TrainingPrice = priceChangeDetails.TrainingPrice
+                TrainingPrice = priceChangeDetails.TrainingPrice,
+                ChangeInitiatedBy = Enum.Parse<ChangeInitiatedBy>(priceChangeDetails.Initiator)
```

Comment:
maybe update this to Initiator / enum called PriceChange Initiator to match elsewhere?

Reply:
Done